examples/diff.c

```diff
@@ -290,9 +290,31 @@ static void parse_opts(struct opts *o, int argc, char *argv[])
 			o->findopts.flags |= GIT_DIFF_FIND_COPIES;
 		else if (!strcmp(a, "--find-copies-harder"))
 			o->findopts.flags |= GIT_DIFF_FIND_COPIES_FROM_UNMODIFIED;
-		else if (is_prefixed(a, "-B") || is_prefixed(a, "--break-rewrites"))
+		else if (is_prefixed(a, "-B") || is_prefixed(a, "--break-rewrites")){
```

Comment:
Please follow the existing style for indentation, braces, etc.  It's roughly the Linux Kernel style:  https://www.kernel.org/doc/Documentation/CodingStyle will be a good introduction.